app/src/main/java/com/eveningoutpost/dexdrip/StartNewSensor.java

```diff
@@ -160,21 +160,45 @@ public void onTimeUpdated(int newmins) {
                     ucalendar.add(Calendar.HOUR_OF_DAY, -1); // hack for warmup time
                 }
 
-                realStartSensor();
+                startSensorOrAskForG6Code();
             }
         });
         timePickerFragment.show(activity.getFragmentManager(), "TimePicker");
     }
 
-    private void realStartSensor() {
+    private void startSensorOrAskForG6Code() {
         if (Ob1G5CollectionService.usingCollector() && Ob1G5StateMachine.usingG6()) {
-            G6CalibrationCodeDialog.ask(this, this::realRealStartSensor);
+            G6CalibrationCodeDialog.ask(this, this::startSensorAndSetIntent);
         } else {
-            realRealStartSensor();
+            startSensorAndSetIntent();
         }
     }
 
+    private void startSensorAndSetIntent() {
+        long startTime = ucalendar.getTime().getTime();
+        Log.d(TAG, "Starting sensor time: " + JoH.dateTimeText(ucalendar.getTime().getTime()));
```

Comment:
I still don't understand why `UserError.Log.x` isn't used all the time.

Reply:
I don't know, Log.d is a standard Android debugging method, right? versus the custom UserError.Log stuff

Reply:
Currently, I don't know whether the standard logging `Log` is intercepted to store the data also in the `UserError` database table. If yes, we don't have to care about writing `Log.x` or `UserError.Log.x` otherwise we have to write `UserError.Log.x` to get it logged to the database as well.

Reply:
I think its possible with the correct import to have `Log.x` use `UserError` class.